.factory/automation.yml

```diff
@@ -32,201 +32,164 @@ build:
       owner: vaticle
       branch: master
     dependency-analysis:
-      image: vaticle-ubuntu-21.04
+      image: vaticle-ubuntu-22.04
       command: |
         bazel run @vaticle_dependencies//factory/analysis:dependency-analysis
   correctness:
     build:
-      image: vaticle-ubuntu-21.04
+      image: vaticle-ubuntu-22.04
       type: foreground
       command: |
-        pyenv install 3.7.12
```

Comment:
We had to make a number of changes to the way we load Python and Pip and install packages in order to work in the new Factory CI machines.